contributors/design-proposals/standardization-of-development-process.md

```diff
@@ -0,0 +1,282 @@
+- a road map can be generated by referring to RFC content
+
+The process for creating draft release notes and development status reports
+should eventually rely largely on automation, but I do not believe that the
+implementation of such automation should prevent the RFC process from being
+considered stable. I can certainly imagine a future where a release notes draft
+is produced nightly along with the collection of binaries that users deploy.
+
+## Drawbacks
+
+Any additional process has the potential to engender resentment within the
+community. There is also a risk that the RFC process as designed will not
+sufficiently address the scaling challenges we face today. PR review bandwidth is
+already at a premium and we may find that the RFC process introduces an unreasonable
+bottleneck on our development velocity.
+
+It certainly can be argued that the lack of a dedicated issue/defect tracker
+beyond GitHub issues contributes to our challenges in managing a project as large
+as Kubernetes, however, given that other large organizations, including GitHub
+itself, make effective use of GitHub issues perhaps the argument is overblown.
+
+The centrality of Git and GitHub within the RFC process also may place too high
+a barrier to potential contributors, however, given that both Git and GitHub are
+required to contribute code changes to Kubernetes today I would argue that we
+should focus on providing support to those unfamiliar with this tooling.
+
+Expanding the proposal template beyond the single sentence description currently
+required in the [features issue template][] may be a heavy burden for non native
+English speakers and here I believe that the role of the RFC editor combined with
+kindness and empathy will be crucial to making the process successful.
+
+[features issue template]: https://github.com/kubernetes/features/blob/master/ISSUE_TEMPLATE.md
+
+## Alternatives
+
+This RFC process is related to
+- the generation of our [architectural road map][]
+- the fact that the [what constitutes a feature][] is still undefined
+- how we [manage issues][]
+- the difference between an [accepted design and a proposal][]
+- [the organization of design proposals][]
+
+and I believe that this RFC process attempts to place these concerns within a
+more general framework
+
+[architectural road map]: https://github.com/kubernetes/community/issues/952
+[what constitutes a feature]: https://github.com/kubernetes/community/issues/531
+[manage issues]: https://github.com/kubernetes/community/issues/580
+[accepted design and a proposal]: https://github.com/kubernetes/community/issues/914
+[the organization of design proposals]: https://github.com/kubernetes/community/issues/918
+
+## Unresolved Questions
+
+Whether we believe it makes sense for a SIG repository to serve a mini mono
+repository where code lives in a structure like
+
+```
+kubernetes:sig-node/pkg/kublet
+```
+
+or whether SIGs maintain a list of repositories maintained by the SIG in an index
```

Comment:
I expect a list of code owned by a SIG and its subprojects to be enumerated in the SIG charter, once we have a template for that.

Reply:
Agree -- the mapping of code/artifact to SIG is a separable issue from this proposal process.  But having a clear mapping there will help to identify when changes are impactful enough to trigger this process.